components/textbox.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from "react";
 import styles from "../styles/Textbox.module.css";
 import Pointsbox from "./pointsbox";
 import { AssessmentType } from "../types/Types";
+import parse from "html-react-parser";
```

Comment:
Looks like this is a continuation of #89 , but it'll probably be fine when we merge this _after_ #89 :)

Reply:
Yup, I branched out from #89